### ISSUE TITLE
"This profile was last used with a newer version of this application."

### ISSUE THREAD

**Issue by TheMaker69420** [kawapure/firefox-native-controls#2]
i have made many new profiles on firefox, and i have added the provided xul file for 115esr to the installation folder so i can run silverfox, and i keep getting this error, despite following its instructions. this has gotten pretty annoying, and i want to continue using silverfox without this error. what am i doing wrong? is there a fix for this? i am on a windows 11 home 64-bit laptop. 
![Screenshot 2024-06-18 121006](https://github.com/florinsdistortedvision/silverfox/assets/134236587/5e58a54a-6792-4a39-99ae-db69d5d99232)

**Comment by kawapure** [comment]
Go to `compatibility.ini` in every profile folder you have (+ `chrome_debugger_profile` subdirectory of each profile folder, if it exists) and set the `LastVersion` entry to 0. This should force all profiles to succeed the check and prevent this error message.

**Comment by fliptether** [comment]
Tried your solution, but it didn't seem to fix it. LastVersion simply resets to the version I'm using and the error will continually pop up at random.

**Comment by kawapure** [comment]
You had not gone through every profile, then.

Alternatively, you can delete all of your Firefox profiles and recreate all of them.

**Comment by fliptether** [comment]
I deleted my profiles and recreated them, aswell as set every profile compatibility.ini to 0 while firefox was closed. It still will randomly show up, not even when I *start* the profile. Sometimes I'll just be sitting at my computer idle while firefox is open, and the popup will show. 

In general, reverting the xul.dll to the one supplied with stock 115.12 seems to be the only way I can fix this issue.
